src/modules/instruction-element/dtos/generic-type.dto.ts

```diff
@@ -0,0 +1,5 @@
+export class GenericTypeDto {
+  id?: number
+  name: string
+  type: string
```

Comment:
maybe the type should be an enum. and contains two options:
PREDEFINED
CUSTOM

and the name has a **InstructionElementGenericType** type